### PR TITLE
contact email

### PR DESCRIPTION
**Pull request by dovyblue**

### CODE REVIEW

**Comment by dovyblue** [comment]
https://trello.com/c/yENfOLJQ/32-as-a-visitor-i-want-to-complete-the-inquiry-page-aka-contact-us-so-that-i-can-provide-my-basic-contact-information-for-being-con

File: Gemfile [hunk 1]
```diff
@@ -31,8 +31,6 @@ gem 'aws-sdk'
 gem 'jquery-rails'
 # Turbolinks makes navigating your web application faster. Read more: https://github.com/turbolinks/turbolinks
 gem 'turbolinks', '~> 5'
-gem 'rails-controller-testing'
```
Comment (trevordjones) [inline Gemfile]: Are you sure you can get rid of this gem? Are other tests using it?

File: app/helpers/pages_helper.rb [hunk 2]
```diff
@@ -1,2 +1,22 @@
...
+       subject: params[:subject],
+       html:  "
+         <p>Name: #{params[:name]}</p>
+         <p>Email: #{params[:email]}</p>
+         <p>Phone Number: #{params[:number]}</p>
+         <h3>Message</h3>
+         <p>#{params[:message]}</p>"
+      }
+
+    result = mg_client.send_message('sandbox4f9920610a894b81b82f6bc37e90f1a0.mailgun.org', message_params).to_h!
```
Comment (trevordjones) [inline app/helpers/pages_helper.rb]: Let's set this domain to an ENV variable so it's not in our code and can be changed based on whatever environment the app is running in

File: app/views/pages/contact.html.erb [hunk 3]
```diff
@@ -80,7 +83,8 @@
                             </div>
                         </div>
                         <!-- Message Input Area End -->
-                    </form>
+                    <% end %>
+                    <!-- </form> -->
```
Comment (trevordjones) [inline app/views/pages/contact.html.erb]: You can remove this commented out form

**Pull request merged**